### ISSUE TITLE
dockerize digital-land-builder

### ISSUE THREAD

**Issue by Ben-Hodgkiss** [digital-land/technical-documentation#95]
TLDR;

Bundled up the current github action logic into a docker image in one centralised repo.

Acceptance Criteria:

- Operating system/co-dependencies to be baked into the image
- Must be runnable for any collection based off of environment variables
- Must be runnable/testable locally
- Run python testing locally. this is a preference for testing using python where possible but it may not be possible.
- Must have a docker compose file which can be used to run digital-land-builder locally
- Must be able to run locally without docker (but with dependencies such as digital-land-python installed)

**Comment by cjohns-scottlogic** [comment]
Blocked pending DevOps changes.

**Comment by CarlosCoelhoSL** [comment]
Q: Second bullet point, 'Must be runnable for any collection based off of environment variables'
What does this mean? Digital land builder is currently not run for any specific collection, it is ran for all collections
